server/container/testing/pom.xml

```diff
@@ -33,6 +33,12 @@
     <name>Apache James :: Server :: Deployment Testing</name>
 
     <dependencies>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>apache-james-backends-cassandra</artifactId>
```

Comment:
this won't work with eclipse : a non-test-jar can't depend on a test-jar anymore

Reply:
Depedency removed in the end...